### DISCUSSION TITLE
Support for time varying parameters

### DISCUSSION THREAD

**Discussion by jradavenport** [phoebe-project/phoebe2#392] [Ideas]
Hello, I am wondering if there is an obvious mechanism in the forward modeling that supports system parameters that are changing with time (e.g. inclination). I see examples in the [tutorials](http://phoebe-project.org/docs/2.3/tutorials) that demonstrate for changing orbital period [dpdt](http://phoebe-project.org/docs/2.3/tutorials/dpdt), but was not clear if such tricks can be played out of the box for other parameters?

**Comment by kecnry** [comment]
Unfortunately, not right now. 

I had at one point implemented a framework for arbitrary time-derivatives of any parameter... but it quickly got out of control and lead to too many difficult cases to handle (if you allow a mass change, for example, what other orbital parameter(s) have to change to keep obeying Kepler's third law).  Inclination should be fairly straightforward though (since it only affects orientation) and is implemented (but disabled) in some parts of the code already.

I could try to take a stab at finishing the implementation for inclination if you'd be interested and willing to test a development branch before it gets merged into the main branches?  Were there any other parameters you think would significantly benefit from time-dependence?
